### ISSUE TITLE
Spacing not supported into float range input

### ISSUE THREAD

**Issue by orhygine** [geopython/pywps#349]
# Description

While defining an input with a range and a spacing like:

```python
LiteralInput('float_input',
             'Input name',
             data_type='float',
             allowed_values=[[0.0, 0.2, 10.0]],
             min_occurs=0)
```
validation failed with following log:

```
2018-06-27 15:03:49,639] [DEBUG] file=.../pywps/validator/literalvalidator.py line=35 module=literalvalidator function=validate_allowed_values validating allowed values: 0.0 in [<pywps.inout.literaltypes.AllowedValue object at 0x7fed2620f450>]
2018-06-27 15:03:49,639] [DEBUG] file=.../pywps/validator/literalvalidator.py line=71 module=literalvalidator function=_validate_range validating range: 0.0 in <pywps.inout.literaltypes.AllowedValue object at 0x7fed2620f450>
2018-06-27 15:03:49,639] [DEBUG] file=.../pywps/validator/literalvalidator.py line=93 module=literalvalidator function=_validate_range validation result: False
2018-06-27 15:03:49,639] [DEBUG] file=.../pywps/validator/literalvalidator.py line=47 module=literalvalidator function=validate_allowed_values validation result: False
2018-06-27 15:03:49,640] [ERROR] file=.../pywps/exceptions.py line=45 module=exceptions function=__init__ Exception: code: 400, locator: Input data not valid using mode 1, description: 
None
```

# Environment

- operating system: ubuntu
- Python version: 2.7
- PyWPS version: master
- source/distribution
 - [x] git clone
 - [ ] Debian
 - [ ] PyPI
 - [ ] zip/tar.gz
 - [ ] other (please specify): 
- web server
 - [x] Apache/mod_wsgi
 - [ ] CGI
 - [ ] other (please specify): 

# Steps to Reproduce

# Additional Information

**Comment by jachym** [comment]
Thanks @cehbrecht Can we close ?

**Comment by cehbrecht** [comment]
@jachym yes, I will close it. Fixed with PR #350 (develop). Merged into master with cherry-pick: 
https://github.com/geopython/pywps/commit/abdd4a3f01404e98d2d1705663537f9c2d1e5552
